unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/Base Layers/RenderBackgroundColorLayer.cs

```diff
@@ -26,16 +26,21 @@ static void RenderHorizonLayer(ref float timeOfTheDay, EditorToolMeasurements to
             EditorGUILayout.Separator();
 
             // Horizon Mask
-            RenderSimpleValues.RenderTexture(SkyboxEditorLiterals.texture, ref selectedConfiguration.horizonMask);
+            EditorGUILayout.LabelField("Horizon Mask", EditorStyles.boldLabel);
+            EditorGUI.indentLevel++;
 
-            // Horizon mask values
-            RenderSimpleValues.RenderVector3Field(SkyboxEditorLiterals.horizonMaskValues, ref selectedConfiguration.horizonMaskValues);
+            RenderSimpleValues.RenderTexture("Texture", ref selectedConfiguration.horizonMask);
```

Comment:
Several of these strings already exist at `unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/SkyboxEditorLiterals.cs`, you could add the missing ones and reuse the ones that are already there